packages/grafana-ui/src/components/uPlot/config/UPlotSeriesBuilder.ts

```diff
@@ -15,6 +15,7 @@ import { PlotConfigBuilder } from '../types';
 export interface SeriesProps extends LineConfig, AreaConfig, PointsConfig {
   drawStyle: DrawStyle;
   scaleKey: string;
+  show?: boolean;
```

Comment:
for the uplot config... rather than requiring an affermative "show", can we do `hide?: boolean`?  this will have better default semantics automatically

Reply:
`!false`